fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -376,22 +378,32 @@ public interface Wait extends StoreTimer.Event {
 
     /**
      * {@link Event}s that only count occurrences or total size.
-     * There is no meaningful time duration associated with these events.
+     * There is no meaningful cumulative value associated with these events.
      */
     public interface Count extends StoreTimer.Event {
         /**
          * Get whether the count value is actually a size in bytes.
          *
          * @return {@code true} if the count value is actually a size in bytes
          */
-        boolean isSize();
+        default boolean isSize() {
+            return false;
+        }
     }
 
     /**
-     * Contains the number of occurrences and cummulative time spent on an associated {@link StoreTimer.Event}.
+     * {@link Event}s that only count the number of occurrences of a read/write operation and also the size read/written.
+     * There is no meaningful time duration associated with these events.
+     */
+    public interface SizeEvent extends StoreTimer.Event {
```

Comment:
There should be an issue associated with this change, and a release notes update.

Reply:
I created an issue: https://github.com/FoundationDB/fdb-record-layer/issues/2525

Reply:
thank you!

Reply:
The release notes still need to be updated, and it would probably be good to have use the `Resolves` syntax in the commit message to auto-resolve the issue.
See: https://github.com/FoundationDB/fdb-record-layer/pull/2510/files#diff-53d9d97604b51916cd0e08d61a4e86d6357b9a672ee28c624df8aca4ec50647dL24 as an example

See also: https://github.com/FoundationDB/fdb-record-layer/blob/main/CONTRIBUTING.md#opening-a-pull-request

Reply:
Sorry, did that now!